tfidf-benchmark/NLP-pipeline-cuML.ipynb

```diff
@@ -0,0 +1,1650 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.                with SimpleTimer() as overalltimer:
</pre><p>We should put this function in a function to make this easy to follow. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>